admin/resources/js/fakeData/fakePools.ts

```diff
@@ -13,7 +13,19 @@ export default (): Pool[] => [
       en: "Autem qui voluptas cumque. Molestiae dolor beatae tempora porro. Dolorem eligendi tempora consectetur id nostrum nihil. Quis et aut sit eveniet voluptatum. Quod veritatis vitae dignissimos provident.",
       fr: "A perferendis ut iste saepe error nihil. Fugiat eum qui et molestiae sapiente aperiam. Dolorum est nemo quasi consequatur ipsam facere fuga. In aliquid corrupti consequatur.",
     },
-    classifications: [],
+    classifications: [
+      {
+        id: "asdf1",
```

Comment:
I believe `id` has to be a **bigint** in the database.
```suggestion
        id: 1,
```